frontend/src/components/board/Sidebar.tsx

```diff
@@ -0,0 +1,92 @@
+import { FC, useState, useRef } from "react";
+import { MenuGroup, MenuItem } from "../../interfaces/canva-interfaces";
+import { CogIcon } from "lucide-react";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+
+import RenderItem from "./SidebarItem";
+
+interface DashboardSidebarProps {
+  menuItem: MenuGroup;
```

Comment:
This prop is called menuItem but is of MenuGroup type?
```suggestion
  menuGroup: MenuGroup;
```